manim/animation/indication.py

```diff
@@ -91,15 +99,15 @@ def create_target(self):
 class Flash(AnimationGroup):
     def __init__(
         self,
-        point,
-        line_length=0.2,
-        num_lines=12,
-        flash_radius=0.3,
-        line_stroke_width=3,
-        color=YELLOW,
-        run_time=1,
+        point: np.ndarray,
+        line_length: typing.Optional[float] = 0.2,
```

Comment:
I thought Optional meant that the type can be float or it can be None. Or does it mean that the argument itself is optional because it has a default value?

Reply:
Yeah. Looks like I have confused it should be just float.